Seohee/Lv2/기능개발.js

```diff
@@ -0,0 +1,33 @@
+function solution(progresses, speeds) {
+    const answer = [];
+
+    // 나머지 작업 시간 담는 배열
+    const restProgresses = progresses.map((progress, index) => {
+        let count = 0;
+        // 100전까지 날짜를 하나씩 증가
+        while (progress < 100) {
+            progress += speeds[index];
+            count++;
+        }
+        return count;
+    })
+    // 처음 원소를 최대값으로 설정한후 반복문 수행
+    let finsh = 1;
+    let max = restProgresses[0];
+    for (let i = 1; i < restProgresses.length; i++) {
+        if (restProgresses[i] <= max)
+            finsh++;
```

Comment:
```suggestion
        if (restProgresses[i] <= max) finsh++;
```
짧으면 한 줄로 표현할 수 있고 줄바꿈이 들어갈 땐 중괄호로 묶어주세요

Reply:
네!!! 주의해서 다음과제 올리겠습니다 !